Tests/SonarQube.TeamBuild.PreProcessor.Tests/RoslynExportProfileTests.cs

```diff
@@ -32,44 +32,55 @@ public void RoslynProfile_LoadValidProfile_Succeeds()
     </RuleSet>
 
     <AdditionalFiles>
-      <AdditionalFile FileName=""SonarLint.xml"" >
-         <AnalysisInput />
-      </AdditionalFile>
-      <AdditionalFile FileName=""MyAnalyzer.xml"" >
-         <Foo />
-      </AdditionalFile>
+      <AdditionalFile FileName=""SonarLint.xml"" >PHRlc3Q+PHN1Yi8+PC90ZXN0</AdditionalFile>
+      <AdditionalFile FileName=""MyAnalyzer.xml"" >PEZvby8+</AdditionalFile>
     </AdditionalFiles>
   </Configuration>
 
   <Deployment>
     <NuGetPackages>
       <NuGetPackage Id=""SonarLint"" Version=""1.3.0""/>
-      <NuGetPackage Id=""My.Analyzers"" Version=""1.0.5.0""/>
+      <NuGetPackage Id=""My.Analyzers"" Version=""1.0.5.0-rc1""/>
     </NuGetPackages>
   </Deployment>
 </RoslynExportProfile>";
 
-            RoslynExportProfile profile = null;
-            using (StringReader reader = new StringReader(validXml))
-            {
-                profile = RoslynExportProfile.Load(reader);
-            }
-
-            Assert.IsNotNull(profile);
-            Assert.IsNotNull(profile.Configuration);
-            Assert.IsNotNull(profile.Configuration.RuleSet);
+            RoslynExportProfile profile = LoadAndCheckXml(validXml);
 
             AssertExpectedAdditionalFileExists("SonarLint.xml", profile);
             AssertExpectedAdditionalFileExists("MyAnalyzer.xml", profile);
 
             AssertExpectedPackageExists("SonarLint", "1.3.0", profile);
-            AssertExpectedPackageExists("My.Analyzers", "1.0.5.0", profile);
+            AssertExpectedPackageExists("My.Analyzers", "1.0.5.0-rc1", profile);
+        }
+
+        [TestMethod]
+        public void RoslynProfile_LoadRealExample_Succeeds()
```

Comment:
Added test to check a real-world example returned from the exported can be deserailized.